README.md

```diff
@@ -2,23 +2,23 @@
 A way for partners to create connectors and destinations that run on [Fivetran’s platform](https://www.fivetran.com/). Read more in our [product docs](https://fivetran.com/docs/partner-built-program). 
 
 ## Repo Structure
-This repo consists of example connectors and destinations along with a local testing environment. Connectors and destinations are written in a [gRPC supported language](https://grpc.io/docs/languages/) that can generate a statically linked binary. We recommend java, golang or rust. 
+This repo consists of example connectors and destinations along with a local testing environment. Connectors and destinations are written in a [gRPC supported language](https://grpc.io/docs/languages/) that can generate a statically linked binary. We recommend Java, Golang, or Rust. 
 * [Examples](examples/)
 * [Local Testing Tools](tools/)
 
 ## Development
 You can follow the [SDK Development Guide](development-guide.md) for guidance on how to develop your code. 
 
 Once you have your code ready to run:
-1. Start up your connector running on port 50051 (for destination code, use port 50052)
-2. Run the local test environment
+1. Start up your connector running on port 50051 (for destination code, use port 50052).
+2. Run the local test environment.
 
-Our Readme's for [connectors](tools/connector-tester/) and [destinations](tools/destination-tester/) will take you through setting up the local test environment. 
+Our READMEs for [connectors](tools/connector-tester/) and [destinations](tools/destination-tester/) will take you through setting up the local test environment. 
 
-> To run an example connector or destination from `examples` directory, follow the Readme specific to the example. 
+> To run an example connector or destination from `examples` directory, follow the README specific to the example. 
 
 ### Other Examples
-You can look at the following public repos to see how other partner's have built their connectors:
+You can look at the following public repos to see how other partners have built their connectors:
 #### Connectors
 * [PlanetScale](https://github.com/planetscale/fivetran-source)
 * [Convex](https://github.com/get-convex/convex-fivetran-source)
```

Comment:
@5tran-alexil this link is outdated
here is the correct link https://github.com/get-convex/convex-backend/tree/main/crates/fivetran_source

Reply:
I have updated it.